analytics/src/docker/Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM jetty:9.3-jre8
 
-ADD . /
-
 RUN java -jar "$JETTY_HOME/start.jar" --add-to-startd=jmx,jmx-remote,stats
 
-CMD ["sh", "-c", "exec java -Djava.io.tmpdir=/tmp/jetty -Dgeorchestra.datadir=/etc/georchestra -Xmx$XMX -Xms$XMX -jar /usr/local/jetty/start.jar"]
+ADD . /
+
+VOLUME [ "/tmp" ]
 
+CMD ["sh", "-c", "exec java \
+-Djava.io.tmpdir=/tmp/jetty \
+-Dgeorchestra.datadir=/etc/georchestra \
+-Xmx$XMX -Xms$XMX \
```

Comment:
You can set a default value, this has been done for gn:
https://github.com/georchestra/geonetwork/blob/georchestra-15.12/web/src/docker/Dockerfile#L21

This to avoid not being able to start the JVM because of missing $XMX env variable.

In addition, that will be the occation to add a $JAVA_OPTIONS in the CMD lines, as discussed this midday.